RecoLocalMuon/GEMSegment/python/me0Segments_cfi.py

```diff
@@ -11,8 +11,8 @@
         dYclusBoxMax = cms.double(5.),
         preClusteringUseChaining = cms.bool(True),
         dPhiChainBoxMax = cms.double(.02),
-        dEtaChainBoxMax = cms.double(.05),
-        dTimeChainBoxMax = cms.double(1.50), # 1ns, +/- time to fly through 30cm thick ME0
+        dEtaChainBoxMax = cms.double(.15),
+        dTimeChainBoxMax = cms.double(32.0), # 1ns, +/- time to fly through 30cm thick ME0
```

Comment:
not exactly, the algorithm looks for close-by hits opening a window around each rechit: https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/RecoLocalMuon/GEMSegment/plugins/ME0SegmentAlgorithm.cc#L280 . When there was no discretization usually the window was set to 3 times the time resolution + tof to me0, in order not to loose hits in the tails. Maybe we can reduce the window to +-15ns, so that in case we switch off the discretization we are in any case fine. To reduce the OOT, we can change the logic and throw away all the recHits that are not in time, and then clusterize the remaining recHits